asmcomp/printcmm.ml

```diff
@@ -61,9 +61,11 @@ let operation d = function
   | Capply _ty -> "app" ^ Debuginfo.to_string d
   | Cextcall(lbl, _ty, _alloc, _) ->
       Printf.sprintf "extcall \"%s\"%s" lbl (Debuginfo.to_string d)
-  | Cload (c, Asttypes.Immutable) -> Printf.sprintf "load %s" (chunk c)
-  | Cload (c, Asttypes.Mutable) -> Printf.sprintf "load_mut %s" (chunk c)
-  | Cloadmut -> "load_mut_rb"
+  | Cload {memory_chunk; mutability} -> (
+    match mutability with
+    | Asttypes.Immutable -> Printf.sprintf "load %s" (chunk memory_chunk)
+    | Asttypes.Mutable   -> Printf.sprintf "load_mut %s" (chunk memory_chunk) )
```

Comment:
It is a good time to change `load_mut` to `load_mut_imm` and `load_mut_rb` to `load_mut_ptr`. I think it is better than `load_mut` and `load_mut_rb`.